.github/workflows/weekly-metrics-discord.yml

```
@@ -0,0 +1,96 @@
+          echo "pr_avg_merge_time=${PR_AVG_TIME_TO_MERGE:-N/A}" >> $GITHUB_OUTPUT
+
+      - name: Send to Discord
+        uses: sarisia/actions-status-discord@v1
+        if: env.DISCORD_WEBHOOK != ''
+        with:
+          webhook: ${{ env.DISCORD_WEBHOOK }}
+          status: Success
+          title: "📊 Weekly Metrics Report"
+          description: |
+            **${{ env.week_of }}**
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)